models/articles-models.js

```diff
@@ -30,7 +30,22 @@ async function fetchAllArticles() {
   });
 }
 
+function incVotesOnArticle(article_id, inc_votes) {
+    return db
+    .query(`UPDATE articles SET votes = votes + $2 WHERE article_id = $1 RETURNING *;`, [article_id, inc_votes])
+    .then(({ rows }) => {
+        if (rows.length === 0) {
+            return Promise.reject({status: 404, msg: 'article id not found'})
+        }
+        return rows[0]
+    })
+    .catch((err) => {
+        return Promise.reject(err)
+    })
```

Comment:
This catch block is redundant and is essentially catching the error that is already occurring in the Promise.reject above. You can remove this 👍